dask_sql/physical/rel/logical/aggregate.py

```diff
@@ -157,6 +157,48 @@ class DaskAggregatePlugin(BaseRelPlugin):
         "single_value": AggregationSpecification("first"),
         # is null was checked earlier, now only need to compute the sum the non null values
         "regr_count": AggregationSpecification("sum", AggregationOnPandas("sum")),
+        "regr_syy": AggregationSpecification(
+            dd.Aggregation(
+                "regr_syy",
+                lambda s: (s.count(), s.sum(), s.agg(lambda x: (x**2).sum())),
+                lambda count, sum, sum_of_squares: (
+                    count.sum(),
+                    sum.sum(),
+                    sum_of_squares.sum(),
+                ),
+                lambda count, sum, sum_of_squares: (
+                    sum_of_squares - (sum * (sum / count))
+                ),
+            )
+        ),
+        "regr_sxx": AggregationSpecification(
+            dd.Aggregation(
+                "regr_sxx",
```

Comment:
`regr_syy` and `regr_sxx` currently just output rows with 0.0, which I suspect has something to do with the code here not properly handling two inputs. From [this site](https://docs.teradata.com/r/kmuOwjp1zEYg98JsB8fu_A/ZsXbiMrja5EYTft42VoiTQ), it looks like I only need the first parameter of the function, but I'm having trouble formatting the aggregation with `dd.Aggregation`